moskito-core/src/main/java/net/anotheria/moskito/core/predefined/OSStats.java

```diff
@@ -87,20 +87,18 @@ public OSStats(String aName,  Interval[] selectedIntervals){
 
 	@Override
 	public String toStatsString(String intervalName, TimeUnit unit) {
```

Comment:
StringBuilder leads to less readable code here, since less characters are needed here. It's not about performance optimization.